.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,6 +1,11 @@
 blank_issues_enabled: false
 contact_links:
-  - name: Join Our Slack Channel wg-manifests.
+  - name: Join Our CNCF Slack Channel kubeflow-platform..
+    url: https://app.slack.com/client/T08PSQ7BQ/C073W572LA2
+    about: Join our CNCF Slack Channel.
+  - name: Kubeflow Community.
     url: https://www.kubeflow.org/docs/about/community/
-    about: Join our slack channel and access our meeting schedule.
+    about: This link contains our meeting schedule.
```

Comment:
about: Join our slack channel and access our meeting schedule was correct. now it is not clear how to join the slack channel